astropy/stats/tests/test_sigma_clipping.py

```diff
@@ -164,9 +164,15 @@ def test_invalid_sigma_clip():
     data[3, 4] = np.nan
     data[1, 1] = np.inf
 
+    data_ma = np.ma.MaskedArray(data)
+
     with pytest.warns(AstropyUserWarning,
                       match=r'Input data contains invalid values'):
         result = sigma_clip(data)
+        result_ma = sigma_clip(data_ma)
```

Comment:
I think I'd put this in a separate `with pytest.warns` stanza, otherwise we would not capture if just one of the two raised the warning (alternatively, count the number of warnings).

Reply:
Yes, thanks!